pytissueoptics/scene/examples/example4.py

```diff
@@ -0,0 +1,30 @@
+TITLE = "Lidar Scene"
```

Comment:
Doesn't run. Probably just the logger.logPoint requireing a key. But not sure I would really call that a LiDAR simulation without proper illumination model (blinn-phong). Also a bit out of this PR scope.

Reply:
i probably included that by error